core/include/mutex.h

```diff
@@ -116,6 +120,13 @@ static inline void mutex_lock(mutex_t *mutex)
 /**
  * @brief Unlocks the mutex.
  *
+ * If there is more than one thread waiting for the mutex, the first one that
```

Comment:
See above.

Reply:
Correct would be: "Should there be other threads be waiting on the mutex, the mutex will stay locked and the highest-priority thread that has waited the longest will be woken up."